eng/pipelines/templates/steps/npm-release-task.yml

```diff
@@ -0,0 +1,56 @@
+parameters:
+  Registry: ''
+
+steps:
+  - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+  - ${{if eq(parameters.Registry, 'https://registry.npmjs.org/')}}:
+      - task: PowerShell@2
+        displayName: Get package list
+        inputs:
+          targetType: filePath
+          filePath: eng/tools/publish-to-npm.ps1
+          arguments: -pathToArtifacts $(Package.Archive) -accessLevel "public" -tag "$(Tag)" -additionalTag "$(AdditionalTag)" -registry ${{parameters.Registry}} -npmToken $(azure-sdk-npm-token)
+          pwsh: true
+        condition: succeeded()
+      - ${{ each package in split(variables.NpmPublishList, ',') }}:
+          - pwsh: write-host ${{package}}; write-host ${{variables.NpmPublishList}}
+          - task: EsrpRelease@6
```

Comment:
Does the EsrpRelease support a set of packages from a folder? If so lets do them in batch instead of in a loop like this.

Reply:
but we do want the feature of skipping some of the packages like we used to in the publish-to-npm.ps1 has right? (e.g. diffing and only publish if there's diff between this and the previous package)

Reply:
We could still use the script to produce a directory of packages we want to publish and then publish that set all at once via Esrp